yarn.lock

```diff
@@ -9707,10 +9707,10 @@ htmlparser2@^7.0:
     domutils "^2.8.0"
     entities "^3.0.1"
 
-http-aws-es@6.0.0:
-  version "6.0.0"
-  resolved "https://registry.yarnpkg.com/http-aws-es/-/http-aws-es-6.0.0.tgz#1528978d2bee718b8732dcdced0856efa747aeff"
-  integrity sha512-g+qp7J110/m4aHrR3iit4akAlnW0UljZ6oTq/rCcbsI8KP9x+95vqUtx49M2XQ2JMpwJio3B6gDYx+E8WDxqiA==
+"http-aws-es@npm:@zhongnansu/http-aws-es@6.0.1":
```

Comment:
Are we going to publish @zhongnansu/http-aws-es?  :D

Reply:
yes that's the solution before https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2220 is completed